src/pudl/metadata/resources/eia923.py

```diff
@@ -3,6 +3,15 @@
 from typing import Any
 
 TABLE_DESCRIPTIONS: dict[str, str] = {
+    "_core_eia923__cooling_system_information": (
+        """EIA-923 Cooling System Information, from EIA-923 Schedule 8D.
+
+Reports monthly information about cooling systems at generation facilities,
+mainly water volumes and temperatures. In 2008 and 2009, EIA only reports
+annual averages, but in later years all data is monthly.
+
+Currently in an intermediate state, use at your own risk."""
```

Comment:
See the warning I wrote up in #3403, I think it's possibly clearer than "use at your own risk" :)